src/mount/mount-tool.c

```diff
@@ -152,7 +156,10 @@ static int parse_argv(int argc, char *argv[]) {
         assert(argc >= 0);
         assert(argv);
 
-        while ((c = getopt_long(argc, argv, "hqH:M:t:o:p:A", options, NULL)) >= 0)
+        if (strstr(program_invocation_short_name, "systemd-umount"))
+                        arg_action = ACTION_UMOUNT;
+
+        while ((c = getopt_long(argc, argv, "hqH:M:t:o:p:A:u", options, NULL)) >= 0)
```

Comment:
Sorry, didn't know how getopt worked exactly. Thanks for pointing that out.

>If such a character is followed by a colon, the option requires an argument

https://linux.die.net/man/3/getopt_long